// error.ts
export const main = (error: Error) => {
  console.log('An error has occurred: ', error);
};
